winui/Rating/Clear-rating.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Clear rating value in WinUI Rating control | Syncfusion
```

Comment:
Title: Clear rating value in WinUI Rating control | Syncfusion

Reply:
We have follow the small letter for all other controls.